docker/test/docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
         ipv4_address: 192.168.199.12
 
 
-  orbs-network-node-1:
+  orbs-app-chain-node-1:
```

Comment:
this file has become long and repetitive; please see if we can simplify it (separate PR)

Reply:
opened #1318